docs/package.json

```diff
@@ -5,17 +5,25 @@
   "scripts": {
     "start": "yarn run pre-build && docusaurus start",
     "pre-build": "yarn run copy-md-files && yarn run variables && yarn run program-outputs && yarn run included-sources",
+    "serve": "serve",
```

Comment:
should we add a Makefile target named `livedocs-production` that does:
1. `make docs`
2. `cd docs/ && yarn serve`
3. `open http://localhost:5000` in browser?

Reply:
We could do that yes—I guess `make docs` triggers the `yarn build` command?—the name `livedocs-production` might be a little confusing though 🤔, FWIW Gatsby's convention for this is `develop` vs `build && serve`

Reply:
yep `make docs` runs `build`. `livedocs-production` is confusing you're right (since nothing is really live). The thing is that these convention are a bit too generic to be Makefile targets. One thing we could do is use an ENV variable (true by default) that will control the `make docs` behaviour. For instance, name it `SERVE_DOCS`, in CI we would do `SERVE_DOCS=0 make docs` and locally we would just run `make docs`. WDYT?

Reply:
I don't think the `serve` part from `netlify-cli` should be part of any CI workflow, it's only meant to mirror netlify's server logic locally so you can test functions and redirects and other things. The current CI build workflow should remain, it's only affected by these changes in so far as I created a `_redirects` file under `static` that will get deployed with the site because it will be copied to `build`, and this will have the same effect on netlify's real servers as it has locally with `netlify-cli`—hopefully 🤞

Reply:
Similar to the Gatsby convention I mentioned above: `gatsby serve` is a command that is only used on local machines, it's just for checking whether your build worked